evennia/commands/default/building.py

```diff
@@ -956,18 +955,21 @@ def func(self):
 
             string = ""
             if not obj.destination:
-                string += "Note: %s(%s) did not have a destination set before. Make sure you linked the right thing." % (obj.name, obj.dbref)
+                string += "Note: %s(%s) did not have a destination set before." \
+                          " Make sure you linked the right thing." % (obj.name, obj.dbref)
             if "twoway" in self.switches:
                 if not (target.location and obj.location):
                     string = "To create a two-way link, %s and %s must both have a location" % (obj, target)
                     string += " (i.e. they cannot be rooms, but should be exits)."
                     self.caller.msg(string)
                     return
                 if not target.destination:
-                    string += "\nNote: %s(%s) did not have a destination set before. Make sure you linked the right thing." % (target.name, target.dbref)
+                    string += "\nNote: %s(%s) did not have a destination set before." \
+                              " Make sure you linked the right thing." % (target.name, target.dbref)
```

Comment:
You might want to extract this and the one above (L958) into a separate variable, then `%` interpolate them here with the right values. For example:

```py
new_destination_string = "..."
# ...
string += new_destination_string % (target.name, target.dbref)

Reply:
Thanks Jed.  I get the "Make sure you linked the right thing." message rather often since building by script or by hand, I might create the exit then link it later. Oddly, the exit is displayed as a room object until linked, a sort of reminder to link it somewhere eventually.